coffee/base.py

```diff
@@ -967,3 +1013,26 @@ def __eq__(self, other):
 DEC = Access("DEC")
 IMUL = Access("IMUL")
 IDIV = Access("IDIV")
+Access._modes = [READ, WRITE, RW, INC, DEC, IMUL, IDIV]
+
+
+# Scope of a declaration ##
+
+class Scope(object):
```

Comment:
Abstraction.
Incidentally, you could raise a similar comment for Access Modes in PyOP2